cirq-core/cirq/ops/raw_types_test.py

```diff
@@ -127,13 +127,13 @@ def test_gate():
 
 
 def test_op():
-    a, b, c = cirq.LineQubit.range(3)
+    a, b, c, d = cirq.LineQubit.range(4)
     g = ValiGate()
-    op = g(a)
-    assert op.controlled_by() is op
```

Comment:
Why has this test changed what it asserts? The method no longer returns the same value when adding zero controls?

Reply:
It returns the same value but didn't return `self` because in the new implementation of `controlled_by` on `GateOperation`, I directly forwarded the request to the underlying `gate` instead of having an explicit check of the form.
```python
    if len(control_qubits) == 0:
        return self
```

Added the check and reverted the test from `==` to `is`.